src/main/java/net/dv8tion/jda/internal/requests/restaction/interactions/InteractionCallbackImpl.java

```diff
@@ -36,6 +34,39 @@ public InteractionCallbackImpl(InteractionImpl interaction)
     {
         super(interaction.getJDA(),  Route.Interactions.CALLBACK.compile(interaction.getId(), interaction.getToken()));
         this.interaction = interaction;
+        setErrorMapper(this::handleUnknownInteraction);
+    }
+
+    private Throwable handleUnknownInteraction(Response response, Request<?> request, ErrorResponseException exception)
+    {
+        // While this error response has existed since at least 2022 (https://github.com/discord/discord-api-docs/pull/4484),
+        // Discord does not always report this error correctly and instead sends a 'UNKNOWN_INTERACTION'.
+        // That's why we also have a similar exception at the end of this method.
+        if (exception.getErrorResponse() == ErrorResponse.INTERACTION_ALREADY_ACKNOWLEDGED)
+            return ErrorResponseException.create(
+                    "This interaction was acknowledged by another bot instance\n" +
```

Comment:
I think "bot instance" isn't clear enough wording for this. Probably best to just spell it out completely:

> This interaction seems to have been acknowledged by another process running for the same bot.
To resolve this, try stopping all current processes for the bot that could be responsible, or resetting your bot token.
You can reset your token at ...